user-contrib/Ltac2/g_ltac2.mlg

```diff
@@ -74,8 +74,14 @@ let test_lpar_id_rpar =
   end
 
 let test_ampersand_ident =
-  entry_of_lookahead "test_ampersand_ident" begin
-    lk_kw "&" >> lk_ident
+  Pcoq.Entry.of_parser "test_ampersand_ident" begin fun tok strm ->
+  match Stream.npeek 2 strm with
+  | (KEYWORD "&" | IDENT "&") :: IDENT s :: [] ->
```

Comment:
Side remark: You could use `Tok.match_pattern` on `PIDENT "&"` to centralize in one place the inclusion `KEYWORD "&"` < `IDENT "&"`.

Reply:
I'm not sure '&' can ever be lexed as an IDENT, can it?

Reply:
Hum, indeed...